config/database.yml

```diff
@@ -63,7 +63,7 @@ test:
   database: decide_test
   username: postgres
   password:
-  host: db
+  host: localhost
 
```

Comment:
Don't commit changes to database.yml. This can break development environment for other users. This is a must fix!